docs/source/examples/demo_transformer_model_train_save_upload_to_openSearch.ipynb

```diff
@@ -23,12 +23,27 @@
    "metadata": {},
    "source": [
     "## Step 0: Import packages and set up client\n",
-    "Install `opensearchpy` and `opensearch_py_ml` through pypi"
+    "Install required packages for opensearch_py_ml.sentence_transformer_model\n",
+    "Install `opensearchpy` and `opensearch_py_ml` through pypi\n",
+    "\n",
+    "Please refer https://pytorch.org/ to proper install torch based on your environment setting.  "
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 1,
+   "id": "17a3e085",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# pip install pandas matplotlib numpy torch accelerate sentence_transformers tqdm transformers opensearch-py opensearch_py_ml "
```

Comment:
How about we keep consistent with https://pypi.org/project/opensearch-py-ml/ by using `pip install opensearch-py-ml` ? Seems that can reduce some confusion.

Reply:
sure